### ISSUE TITLE
Can't configure keyboard on launch

### ISSUE THREAD

**Issue by kdawg39** [Aloshi/EmulationStation#327]
Sorry if this is a noob question, but I've searched and haven't found the answer. I Downloaded and installed via the retropie binary. 

Launching brings up the configuration screen. It detects my wireless USB keyboard fine and lets  me assign the various keys. But at the end I can't select 'ok'. The only input that does anything is 'f4' which just quits - therefore I'm stuck. 

Any idea how to resolve this?

**Comment by Aloshi** [comment]
Pressing whatever button you set up as "A" doesn't work?

**Comment by pdigioia** [comment]
I'd suggest improving this UX a bit. It's not immediately clear that the button that the user has configured as "A" is the new "select this option" button. This results in frustration. Folks are used to the enter key just working. Once past this step, the UI includes visual hints (A = Select, Start = Menu, etc) which help a lot. However, this legend doesn't appear in the Configuration screen. Consider allowing the Enter key just for this one part of the workflow of clicking on "OK", or providing a visual hint.

**Comment by androidand** [comment]
I agree, the UI/UX should be improved by letting the user pass the keyboard/input configuration with Enter or hint that the "A" key should be pressed.

**Comment by onebeartoe** [comment]
+1 on the legend at the keyboard config screen

I was just a first time user of that screen and had no idea how to click the okay button.

Luckily I found this and the reddit page on this, but not knowing how to get passed the keyboard config screen might turn off a non-technical person/parent.

**Comment by X-Arcade** [comment]
This is an issue for X-Arcade users too because you can't easily enable Xarcade2Jstick without doing some command line stuff: https://support.xgaming.com/solution/articles/5000554993-how-to-use-x-arcade-with-raspberry-pi 

It would be great if keyboard/X-Arcade users had a way to bypass the gamepad setup entirely so they could at least set up that.
